.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 *.log
 **/*-bundle.js
+dist/*
+!dist/.gitkeep
```

Comment:
Won't these new lines cause the built files to be ignored by npm, so they won't show up in unpkg.com?

Reply:
I forgot that `npm` actually respects `.gitignore` if no `.npmignore` is found and/or `package.json`'s `files` property is not defined. Bottom line, you are right but I'm not sure whether to whitelist stuff using `files` or blacklist using `.npmignore`. Any preferences? :thinking:

Reply:
_Answering to myself:_ You already used `files` so I just extended it...